content/episodes/2019-01-21-S04E04-everything-in-moderation.md

```diff
@@ -0,0 +1,228 @@
+date: 2019-01-21T04:55:00-08:00
+title: "Everything in Moderation"
+ogtitle: "The Human Costs of Content Moderation"
+description: |
+    What, if anything, should be banned from online media? And who should review violent and explicit content, in order to decide if it’s okay for the public? Thousands of people around the world are working long, difficult hours as content moderators in support of sites like Facebook, Twitter, and YouTube. They are guided by complex and shifting guidelines, and their work can sometimes lead to psychological trauma. But the practice of content moderation also raises questions about censorship and free expression online.
+
+    In this IRL episode, host [Manoush Zomorodi](https://twitter.com/manoushz/) talks with a forensic investigator who compares the work she does solving disturbing crimes with the work done by content moderators. We hear the stories of content moderators working in the Philippines, as told by the directors of a new documentary called [The Cleaners](http://www.pbs.org/independentlens/films/the-cleaners/). [Ellen Silver](https://www.linkedin.com/in/silverellen) from Facebook joins us to outline Facebook's content moderation policies. [Kalev Leetaru](https://www.kalevleetaru.com/) flags the risks that come from relying on artificial intelligence to clean the web. And [Kat Lo](https://twitter.com/lawlkat?lang=en) explains why this work is impossible to get exactly right.
+
+    Some of the content in this episode is sensitive, and may be difficult to hear for some listeners.
+ogdescription: "Meet the people responsible for cleaning your social media feed in Mozilla’s IRL podcast."
+number: 27
+season: 4
+seasonepisode: 4
+url: /season4/episode4/
+embed: "2f129d69"
+mp3: "https://dts.podtrac.com/redirect.mp3/audio.simplecast.com/2f129d69.mp3"
+categories: "episodes"
+host: "Manoushz"
+shownotes: |
+    Read the New York Times article on [Facebook's content moderation policies](https://www.nytimes.com/2018/12/27/world/facebook-moderators.html) and also [Facebook’s response](https://newsroom.fb.com/news/2018/12/content-review-facts/).
+
+    Want more? Mozilla has teamed up with [826 Valencia](https://826valencia.org/) to bring you perspectives written by students on IRL topics this season. Nicole M. from [De Marillac Academy](https://demarillac.org/) wrote [this piece on inappropriate content online](http://blog.mozilla.org/internetcitizen/2019/01/20/inappropriate-content-on-the-internet/).
+
+    And, check out [this article from Common Sense Media, on disturbing YouTube videos that are supposed to be for kids](http://blog.mozilla.org/internetcitizen/2019/01/20/whats-the-deal-with-those-disturbing-youtube-videos-that-are-supposed-to-be-for-kids/).
+
+    And finally, this IRL episode’s content underscores the importance of supporting companies committed to ethical tech and humane practices. Thank you for supporting Mozilla by choosing [Firefox](https://www.mozilla.org/en-US/firefox/fights-for-you/).
```

Comment:
I might suggest removing the hardcoded `en-US` from this URL. I know the page is English-only for now but maybe it won't always be?

Reply:
It's also in German!

Reply:
Oh that's right! In that case I'm glad I caught this 😁